actor/builtin/paymentbroker/paymentbroker.go

```diff
@@ -205,7 +207,11 @@ func (pb *Actor) CreateChannel(vmctx exec.VMContext, target address.Address, eol
 // target Redeem(200)          -> Payer: 1000, Target: 200, Channel: 800
 // target Close(500)           -> Payer: 1500, Target: 500, Channel: 0
 //
-func (pb *Actor) Redeem(vmctx exec.VMContext, payer address.Address, chid *types.ChannelID, amt *types.AttoFIL, validAt *types.BlockHeight, condition *types.Predicate, sig []byte) (uint8, error) {
+// If a condition is provided in the voucher:
+// - The parameters provided in the condition will be combined with redeemerSuppliedParams
+// - A message will be sent to the the condition.To address using the condition.Method with the combined params
+// - If the message returns an error the condition is considered to be false and the redeem will fail
+func (pb *Actor) Redeem(vmctx exec.VMContext, payer address.Address, chid *types.ChannelID, amt *types.AttoFIL, validAt *types.BlockHeight, condition *types.Predicate, sig []byte, redeemerSuppliedParams []interface{}) (uint8, error) {
```

Comment:
The name of `redeemerSuppliedParams` could be improved.  The new comment goes a long way at explaining how they are used but IMO the name itself should specify what these are the params of.  `redeemerWitness` (or `redeemerProof` but proof is too overloaded).  If you want to keep params we could be more specific `redeemerSuppliedConditionCheckingParams`, or maybe `redeemerConditionWitness`.